src/spatio_temporal_voxel_grid.cpp

```diff
@@ -173,6 +175,7 @@ void SpatioTemporalVoxelGrid::TemporalClearAndGenerateCostmap(                \
   for (cit_grid; cit_grid.test(); ++cit_grid)
   {
     const openvdb::Coord pt_index(cit_grid.getCoord());
+    const openvdb::Vec3d pose_world = _grid->indexToWorld(pt_index);
```

Comment:
Use the `this` version of this function

Reply:
Ok. I copied the syntax from 

https://github.com/SteveMacenski/spatio_temporal_voxel_layer/blob/e077795c89a280ffffa83db61af540464e34a892/src/spatio_temporal_voxel_grid.cpp#L240-L245

Do you want me to replace it there as well?

Reply:
Yes please

Reply:
Fixed